src/components/side_panel/chart/building_blocks/general_design/general_design_editor.xml

```diff
@@ -3,11 +3,14 @@
     <t t-set="chart_title">Chart title</t>
     <SidePanelCollapsible collapsedAtInit="false" title.translate="General">
       <t t-set-slot="content">
-        <Section class="'o-chart-background-color pt-0 pb-0'" title.translate="Background color">
-          <RoundColorPicker
-            currentColor="props.definition.background"
-            onColorPicked.bind="updateBackgroundColor"
-          />
+        <Section class="'o-chart-background-color pt-0 pb-0'">
+          <div class="d-flex align-items-center">
+            <span class="o-section-title mb-0 pe-2">Background color</span>
+            <RoundColorPicker
+              currentColor="props.definition.background"
+              onColorPicked.bind="updateBackgroundColor"
+            />
+          </div>
```

Comment:
If we do it, we it should be in a separate commit (if not a different task) since it has nothing to do with the "all series editable" feature.

And I'd agree with Adrien. It'd keep it as the designers proposed

Reply:
Up, revert this :wink: